go/libkb/id_table.go

```diff
@@ -453,6 +453,19 @@ func (l *TrackChainLink) GetTrackedKeys() ([]TrackedKey, error) {
 
 	var res []TrackedKey
 
+	// If the eldest kid in this tracking statement is a PGP key, it's
+	// implicitly tracked, in addition to the "pgp_keys" section that follows.
+	eldestKeyJSON := l.payloadJSON.AtPath("body.track.key")
+	eldestTrackedKey, err := trackedKeyFromJSON(eldestKeyJSON)
+	if err != nil {
+		return nil, err
+	}
+	var emptyPGPFingerprint PGPFingerprint
+	if !eldestTrackedKey.Fingerprint.Eq(emptyPGPFingerprint) {
```

Comment:
is this the best way to check for non-PGP-ism?  I don't think I've ever done this before.